Doc/library/profile.rst

```diff
@@ -272,6 +272,8 @@ functions:
 
       pr.print_stats()
 
+   .. versionadded:: 3.8
```

Comment:
Since this applies to the class definition this isn't accurate since the class isn't new 😉 . I would change this to:
```restructuredtext
.. versionchanged:: 3.8
   Added context manager support.
```